tests/test_tools.py

```diff
@@ -224,7 +224,7 @@ def test_replace_members_with_ids(self, client, test_client):
             writers=['~Super_User1'],
             signatures=['~Super_User1'],
             signatories=['~Super_User1'],
-            members=['test@mail.com', '~SomeFirstName_User1', '~Another_Name1']
+            members=['test@mail.com', '~SomeFirstName_User1', '~Another_Name1', 'random_member']
```

Comment:
Can you test this using a group id with capital letters like 'ICLR.cc/2021/Conference/Reviewers'? this group id shouldn't change after the replacement.

Reply:
Do you want to keep it as a member of the group?

Reply:
I updated it to check for the group if it's not an email or a tilde ID. But `get_group()` returns a group for an invalid ID. Is that expected?

Reply:
> Do you want to keep it as a member of the group?
Yes, please

> get_group() returns a group for an invalid ID. Is that expected?
if the group doesn't exist then it will throw an error but all the members of a group should exist. If you get an exception then you can print it out but don't remove it so the PCs can fix this.